### ISSUE TITLE
We have no documentation and no tests

### ISSUE THREAD

**Issue by BlacklightShining** [AppleDash/LunaBot#1]
A good project should have docs and tests, right? Right? >.>

**Comment by AppleDash** [comment]
Agreed. Not really sure how you unit test something like an IRC bot, though.

**Comment by BlacklightShining** [comment]
Very carefully.

…you unit test the obvious things and use mocks for the rest.
